tests/ts/lsfd/column-xmode

```diff
@@ -142,6 +143,10 @@ for m in flock-sh posix-r- ofd-r-; do
 	fi
     } > "$TS_OUTPUT" 2>&1
     wait "${MKFDS_PID}"
+    if [ "$?" == "$ENOSYS" ]; then
+	ts_skip_subtest	"$m lock is not available"
```

Comment:
Why not `EXIT_NOTSUPP` and `ts_skip_exitcode_not_supported`?
Maybe with a new function to only skip the subtests.

Reply:
Thank you. Updated.
I didn't introduce such a new function.
A skip function for subtests cannot exit because more subtests may exist.